doc/parallel_numpy.rst

```diff
@@ -23,6 +23,12 @@ worker processes.
   your code can release the GIL, then using ``backend="threading"`` is even
   more efficient.
 
+..
+  >>> # For running doctests, users may ignore this block
+  >>> import sys
+  >>> setup = getfixture('parallel_numpy_fixture')
+  >>> fixture = setup(sys.modules[__name__])
```

Comment:
Hmm is this the way to do it with py.test? This is slightly cumbersome but fair enough.